scripts/cargo-install-all.sh

```diff
@@ -142,12 +142,12 @@ mkdir -p "$installDir/bin"
 
   # Exclude `spl-token` binary for net.sh builds
   if [[ -z "$validatorOnly" ]]; then
-    # the config is work around and --locked is removed until we ship newer spl-token-cli...
+    # the patch-related configs are needed for rust 1.69+ on Windows; see Cargo.toml
     # shellcheck disable=SC2086 # Don't want to double quote $rust_version
     "$cargo" $maybeRustVersion \
       --config 'patch.crates-io.ntapi.git="https://github.com/solana-labs/ntapi"' \
-      --config 'patch.crates-io.ntapi.rev="5980bbab2e0501a8100eb88c12222d664ccb3a0a"' \
-      install spl-token-cli --root "$installDir"
+      --config 'patch.crates-io.ntapi.rev="97ede981a1777883ff86d142b75024b023f04fad"' \
+      install --locked spl-token-cli --root "$installDir"
```

Comment:
@yihau as you suggested, we can switch from building to downloading some existing spl tarball from somewhere. seems solang is taking this approach as shown below this file. I have no particular preference; but seeing the below code, needs some work for factoring the below download logic, etc (including on Windows).

So, overall, i think `[patch]-ing` is acceptable workaround for now. (after all, this won't be needed, once tokio is upgraded).

Reply:
also, this is a bit too theoretical. but this bundled `spl-token` binary is used rather widely across various integrations. they seldom use `cargo install ...`. so, i think using the same monorepo-pinned rust version is conceptually easier to reason about. admittedly, it's rare for rust to expose quite visible runtime/compile differences between versions. but this is possibility.

Also, it's more easy to ship new binary once rust found to contain critical security vuln (this again very unlikely)

Reply:
sure. let's do this!